docs/IE_DG/Graph_debug_capabilities.md

```diff
@@ -15,8 +15,7 @@ To visualize the nGraph function to the xDot format or to an image file, use the
 
 std::shared_ptr<ngraph::Function> nGraph;
 ...
-std::vector<std::shared_ptr<ngraph::Function>> g2{nGraph};
-ngraph::pass::VisualizeTree("after.png").run_on_module(g2);     // Visualize the nGraph function to an image
+ngraph::pass::VisualizeTree("after.png").run_on_function(nGraph);     // Visualize the nGraph function to an image
```

Comment:
@GlebKazantaev What if I want to serialize the constant nGraph function?

Should I use dynamic cast?

Reply:
run_on_function takes const reference, so there is no need to cast